doc/ref/configuration/minion.rst

```diff
@@ -752,6 +752,30 @@ Statically assigns grains to the minion.
       cabinet: 13
       cab_u: 14-15
 
+.. conf_minion:: grains_blacklist
+
+``grains_blacklist``
+----------
+
+Default: ``[]``
+
+If :conf_minion:`grains_blacklist` is enabled then any core grains will be
+checked against the matches found in the `grains_blacklist` list and any
+matched core grains will be filtered.
```

Comment:
We don't need to use the `:conf_minion:` Sphinx role here, linking to the same config option is redundant. We also should mention that globs and regex matches are both supported. How about this?

```
Each grains key will be compared against each of the expressions in this list.
Any keys which match will be filtered from the grains. Exact matches, glob
matches, and regular expressions are supported.
```